flatpak.plugin.zsh

```diff
@@ -1,4 +1,4 @@
-alias fti="flatpak install"
+#alias fti="flatpak install"
```

Comment:
any reason for this change? if you want to remove it, please do so in a separate commit

Reply:
Right. That wasn't meant to be part of the commit. Removed.